@xen-orchestra/web-core/lib/components/console/VtsRemoteConsole.vue

```diff
@@ -84,6 +84,16 @@ async function createVncConnection() {
 
   vncClient.addEventListener('disconnect', handleDisconnectionEvent)
   vncClient.addEventListener('connect', handleConnectionEvent)
+  const canvas = consoleContainer.value?.querySelector('canvas') as HTMLCanvasElement | null
+  if (canvas !== null) {
```

Comment:
You can add this to have possibility to focus console with tab, what do you think ? 
```suggestion
  if (canvas !== null) {
    canvas.setAttribute('tabindex', '0')
```

Reply:
@pdonias what do you think about it?

Reply:
This is interesting if we want to be able to focus the console without using the mouse. But I think we'll need to discuss it with @clemencebx to see what would be the best way to do it. I think we'd need to define a key to enter the console and a key to exit the console. But maybe tab-focusing the console and pressing Enter would be enough to enter the console. So for now, you can just leave a comment until we decide what to do.

Reply:
@pdonias I can leave a comment on figma .
as for this task, do we validate it with a focus via a mouse click?

Reply:
For now it's ok to only focus it with a mouse click. As for the `tabindex`, you can add it or not, as you want. But what I'd like to add though is a comment in the code so that we remember what we're planning to do.